packages/flutter/lib/src/widgets/icon_theme_data.dart

```diff
@@ -126,6 +117,22 @@ class IconThemeData with Diagnosticable {
     && opacity != null
     && applyTextScaling != null;
 
+  /// Returns a [ConcreteIconThemeData] with null properties set to the
+  /// corresponding default values in [fallback].
+  ConcreteIconThemeData get concreteIconThemeData {
```

Comment:
This isn't just returning a `concreteThemeData`, it's returning one with defaults. Maybe call it `copyWithDefaults`? If that's too close to `copyWith` then maybe just `withDefaults` or `applyDefaults`?